.build-test-rules.yml

```diff
@@ -5,10 +5,16 @@ examples:
     - if: IDF_VERSION_MAJOR < 5 and IDF_TARGET in ["esp32c2", "esp32p4", "esp32c5", "esp32c6"]
       reason: Example depends on target, which is supported only for IDF >= 5.0
 
+components/lcd/esp_lcd_gc9503:
+  disable:
+    - if: IDF_VERSION_MAJOR <= 5.0.4 || IDF_VERSION_MAJOR == 5.1.1
+      reason: Supported from version 5.0.5
+
 components/lcd/esp_lcd_gc9503/test_apps:
   disable:
-    - if: IDF_VERSION_MAJOR < 5
-      reason: Component is supported only for IDF >= 5.0
+    - if: IDF_VERSION_MAJOR <= 5.0.4 || IDF_VERSION_MAJOR == 5.1.1
+      reason: Supported from version 5.0.5
```

Comment:
Does this work? Shouldn't it be just something like IDF_VERSION or IDF_VERSION_VAL?

Reply:
Good point, I overlooked it. Fixed, let's wait for CI pass.